lib/config/fileconf.go

```diff
@@ -535,6 +535,35 @@ type Auth struct {
 	// WebIdleTimeout sets global cluster default setting for WebUI client
 	// idle timeouts
 	WebIdleTimeout types.Duration `yaml:"web_idle_timeout,omitempty"`
+
+	// CAKeyParams configures how CA private keys will be created and stored.
+	CAKeyParams *CAKeyParams `yaml:"ca_key_params,omitempty"`
+}
+
+// CAKeyParams configures how CA private keys will be created and stored.
+type CAKeyParams struct {
+	// PKCS11 configures a PKCS#11 HSM to be used for private key generation and
+	// storage.
+	PKCS11 PKCS11 `yaml:"pkcs11,omitempty"`
```

Comment:
Nit: Since this is not a pointer field, "omitempty" doesn't really do anything here.

Reply:
makes sense, removed